tests/end2end/features/thumbnail/test_thumbnaillazy.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Lazy load thumbnails
```

Comment:
Please rename the file to `thumbnaillazy.feature` instead of prepending `test_` :blush: